TundraTD/Assets/Scripts/Level/PauseMode.cs

```diff
@@ -46,7 +47,7 @@ public void SwitchPause()
             Time.timeScale = IsGamePaused ? 0 : 1;
             resumeButton.gameObject.SetActive(IsGamePaused);
             pauseButton.gameObject.SetActive(!IsGamePaused);
-
+            if (IsGamePaused) _soundEffect.Play();
```

Comment:
I suppose we don't need pausemode play anymore because we have a pull request #46

Reply:
Ok, I think so too